applications/endoscopy_tool_tracking/cpp/endoscopy_tool_tracking.yaml

```diff
@@ -14,6 +14,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ---
+application:
```

Comment:
Could you please add a README reference clarifying that these fields are specified by the CLI app runner? https://docs.nvidia.com/holoscan/sdk-user-guide/cli/run_config.html#